### PR TITLE
fix: shema validation

### PR DESCRIPTION
**Pull request by yuberdysheva**

### CODE REVIEW

File: src/schema/index.ts [hunk 1]
```diff
@@ -56,6 +58,91 @@ export const getBlocksCases = (blocks: SchemaBlock) => {
 export function generateDefaultSchema(config?: SchemaCustomConfig) {
     const {cards = {}, blocks = {}, extensions = {}} = config ?? {};
 
+    const blocksValidators = {
```
Comment (gorgeousvlad) [inline src/schema/index.ts:61]: blockValidators

File: src/schema/index.ts [hunk 2]
```diff
@@ -56,6 +58,91 @@ export const getBlocksCases = (blocks: SchemaBlock) => {
...
+        ...SimpleBlock,
+        ...LinkTableBlock,
+        ...PreviewBlock,
+        ...HeaderSliderBlock,
+        ...IconsBlock,
+        ...CardLayoutBlock,
+        ...ContentLayoutBlock,
+    };
+
+    const cardsValidators = {
```
Comment (gorgeousvlad) [inline src/schema/index.ts:84]: cardValidators

File: src/schema/index.ts [hunk 3]
```diff
@@ -56,6 +58,91 @@ export const getBlocksCases = (blocks: SchemaBlock) => {
...
+        'questions-block',
+        'header-block',
+        'banner-block',
+        'companies-block',
+        'media-block',
+        'info-block',
+        'security-block',
+        'table-block',
+        'tabs-block',
+        'simple-block',
```
Comment (gorgeousvlad) [inline src/schema/index.ts:114]: did we delete it?

**Review by gorgeousvlad — CHANGES_REQUESTED** [review]

File: src/schema/index.ts (hunk 3 above)
Comment (yuberdysheva) [reply src/schema/index.ts:114]: not yet

**Review by gorgeousvlad — APPROVED** [review]

**Pull request merged**